pyproject.toml

```diff
@@ -165,6 +165,7 @@ required-imports = ["from __future__ import annotations"]
 docstring-code-format = true
 
 [tool.pytest.ini_options]
+norecursedirs =  ['*.egg', '.*', '_darcs', 'build', 'CVS', 'dist', 'node_modules', 'venv', '{arch}', "_ibis"]
```

Comment:
thanks - should we add `_spark_like` / etc? would `"narwhals/_*"` work?

Reply:
Something like `narwhals/_*` is what I had in mind but didn't check if that would work 🙃

Reply:
🙂.. it seems to work.